README.md

```diff
@@ -1322,6 +1322,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
   - [Pablo Lorenzo](https://github.com/Ll2NZ)
   - [Palak Bhawsar](https://github.com/palakbhawsar98)
+  - [Paras Nagpal] (https://github.com/PARASnagpal99)
```

Comment:
Hey @PARASnagpal99 ,
Please remove the space between the square bracket and parenthesis `] ( -> ](`

Reply:
did it sir , now please accept my pull request .